website/content/docs/security/acl/auth-methods/index.mdx

```diff
@@ -136,3 +136,26 @@ bearer token for a Consul ACL token by using the login process:
 
 For more details about specific auth methods and how to configure them, click
 on the name of the auth method type in the sidebar.
+
+# Token Locality
+
+By default, Auth Methods create ACL tokens with _local_ scope. ACL tokens with
+local scope are only valid in the local datacenter where they were created.
+Auth Methods created in the primary datacenter can be configured to create ACL
+tokens that have global scope; however, there are limitations depending on the
+type of the Auth Method being used. Please refer to the documentation for the
+given Auth Method being used for the details related to token locality:
+
+- [`kubernetes`](/docs/acl/auth-methods/kubernetes#token-locality)
+- [`jwt`](/docs/acl/auth-methods/jwt#token-locality)
+- [`oidc`](/docs/acl/auth-methods/oidc#token-locality)
+
+For cross-datacenter requests that use a token provided by an Auth Method, one
+of the following must be true for requests to be successfully authenticated:
+- the Auth Method is configured with [`-token-locality=global`](/commands/acl/auth-method/create#token-locality)
+- the anonymous token policy in the remote datacenter grants access to read services
```

Comment:
I think using a local token for the target DC is also an option, but I'm not sure if that works for all workflows.

I know we use the anonymous token "workaround" in consul-k8s, but I'm not sure if that's something we want to document as a supported way of doing things. 

If we do want to keep the "anonymous token" docs here, are there other docs we can link to that explain how to update the policy for the anonymous token?

Reply:
I think more accurately you'd have to say that the anonymous token policy has the permissions needed for the request you're making (because in cross-dc calls, the local token is removed from the call and so when it arrives in the other dc it will be authenticated as an anonymous call)